jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/CreateCapabilities.kt

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.deploy
+
+import com.intellij.ui.components.JBCheckBox
+import software.aws.toolkits.resources.message
+
+enum class CreateCapabilities(val capability: String, val text: String, val toolTipText: String, val defaultSet: Boolean) {
+    IAM(
+        "CAPABILITY_IAM",
+        message("cloudformation.capabilities.iam"),
+        message("cloudformation.capabilities.iam.toolTipText"),
+        true
+    ),
+    NAMED_IAM(
+        "CAPABILITY_NAMED_IAM",
+        message("cloudformation.capabilities.named_iam"),
+        message("cloudformation.capabilities.named_iam.toolTipText"),
+        true
+    ),
+    AUTO_EXPAND(
+        "CAPABILITY_AUTO_EXPAND",
```

Comment:
Can we use the actual enum here? (Capability.CAPABILITY_AUTO_EXPAND.value)

Reply:
Or better yet, the enum itself and convert to the value string when we need it?

Reply:
`value` is actually private, but the toString works, changed. This is really the point where we need it so I don't think we need to store anything other than the string.